### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by MyDapitt**
This change simply changes the Debian docker source, changes the source version number of Microsoft's .NET repository, and adds the imagemagick package.

Good luck ~

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM debian:bullseye-slim
+FROM public.ecr.aws/docker/library/debian:bullseye-slim
```
Comment (wdhdev) [inline Dockerfile]: I don't understand why this is needed? The default `debian:bullseye-slim` should work fine?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (MyDapitt) [reply Dockerfile]: It's just an image from AWS, actually there is nothing different about the two images. Just replace it if it's a burden.

Comment (wdhdev) [reply Dockerfile]: @MyDapitt Yeah, I don't really see why this needs to be changed, can you please revert this line.

Comment (MyDapitt) [reply Dockerfile]: Okay, I was changed it.

File: Dockerfile [hunk 2]
```diff
@@ -34,10 +34,7 @@ ENV PATH=$PATH:$JAVA_HOME/bin
 
 # NodeJS
 RUN curl -sL https://deb.nodesource.com/setup_lts.x | bash - \
-    && apt -y install nodejs \
-    && apt -y install ffmpeg \
-    && apt -y install make \
-    && apt -y install build-essential
```
Comment (wdhdev) [inline Dockerfile:40]: Having these on separate lines can be helpful for readability.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (danielpmc) [reply Dockerfile:40]: This wont be approved at this time. Dockerfiles is something end users don't get to see so it doesnt need to be more readable. And having them as seperate commands can also increase build time and install time.

**Review by wdhdev — APPROVED** [review]

**Review by danielpmc — APPROVED** [review]

**Pull request merged**